apps/studio/src/components/PageEditor/constants.ts

```diff
@@ -84,6 +84,7 @@ export const DEFAULT_BLOCKS: Record<
     type: "infocards",
     title: "This is an optional title of the Infocards component",
     subtitle: "This is an optional subtitle for the Infocards component",
+    isCardsWithImages: true,
```

Comment:
do you think there can be other states for cards? if so we can use an enum like "WithImages, TextOnly" etc. and is extensible?

Reply:
Ah good point, this was the original based on designs where there was a toggle button but this has changed to use a dropdown to select the variants. Adjusted in 34dcbfd2de27c92504ddae29f4845f7835d1a8b0!